app/components/custom-input/CustomInput.tsx

```diff
@@ -29,6 +29,7 @@ const CustomInput = React.forwardRef((props: CustomTextInputType, ref) => {
     <>
       <View style={customInputStyles.inputContainer}>
         <TextInput
+          editable={props.editable ?? true}
```

Comment:
have you tried passing editable as false ? and check what happens

Reply:
resolved